test/test_model.py

```diff
@@ -2951,7 +2945,18 @@ def test_local_get_status(self, fake_script: FakeScript, name: str):
     def test_status_set_is_app_not_bool_raises(self):
         for is_app_v in [None, 1, 2.0, 'a', b'beef', object]:
             with pytest.raises(TypeError):
-                self.backend.status_set(ops.ActiveStatus, is_app=is_app_v)  # type: ignore
+                self.backend.status_set(
+                    'active',
+                    is_app=is_app_v,  # type: ignore[assignment]
+                )
+
+    def test_status_set_message_not_str_raises(self):
+        for message in [None, 1, 2.0, True, b'beef', object]:
```

Comment:
My gut tells me you meant:

```suggestion
        for message in [None, 1, 2.0, True, b'beef', object()]:
```

Reply:
To be honest I just copied the test cases from `test_status_set_is_app_not_bool_raises` and changed the `str` one to a `bool`. I'll update them both to avoid future puzzlement.